decisions/wc_offmap_decisions.txt

```diff
@@ -779,4 +779,81 @@ offmap_decisions = {
 			}
 		}
 	}
-}
+	request_healing_offmap = {
+		only_playable = yes
+		button_name = offmap_boons
+
+		ai_check_interval = 24
+
+		potential = {
+			is_old_god_offmap_tag_trigger = yes
+		}
+		from_potential = {
+			offmap_from_potential_trigger = yes
+		}
+		allow = {
+			trigger_if = {
+				limit = { is_offmap_tag = offmap_cthun }
+				FROM = {
+					has_offmap_currency = { offmap = offmap_cthun value = 500 }
```

Comment:
I would increase the cost to 1000-750. It's much better than society's Dark Healing.